clustering/jgroups/extension/src/main/java/org/jboss/as/clustering/jgroups/subsystem/SocketTransportResourceDefinitionRegistrar.java

```diff
@@ -50,6 +56,23 @@ public PathElement getPathElement() {
             .setAccessConstraints(SensitiveTargetAccessConstraintDefinition.SOCKET_BINDING_REF)
             .build();
 
+    private static final String CLIENT_CONTEXT_NAME = "client-context";
```

Comment:
Can we change this to: "client-ssl-context"?

Reply:
We might need to, because these names are appearing in the console.

Context - [#wildfly-developers > Console issues in JGroups subsystem](https://wildfly.zulipchat.com/#narrow/channel/174184-wildfly-developers/topic/Console.20issues.20in.20JGroups.20subsystem/with/562965689)